src/lib/CreateFormMixin.js

```diff
@@ -99,7 +100,11 @@ module.exports = {
     // Global errors
     if (globalErrors) errors = errors.concat(globalErrors);
 
-    return errors.length ? errors : false;
+    if (errors.length && errors[0] !== true) {
```

Comment:
This should be 
```
return errors.length ? errors : true;
```
i.e. "If there are errors in the array, return it. If not, return true, because the field passed all validation tests"